packages/core/src/messagingProtocols/mls/E2EIdentityService/E2EIServiceExternal.ts

```diff
@@ -44,7 +44,7 @@ import {StringifiedQualifiedId, stringifyQualifiedId} from '../../../util/qualif
 import {RecurringTaskScheduler} from '../../../util/RecurringTaskScheduler';
 import {MLSService, MLSServiceEvents} from '../MLSService';
 
-export type DeviceIdentity = Omit<WireIdentity, 'free' | 'status'> & {
+export type DeviceIdentity = Omit<WireIdentity, 'free' | 'status' | typeof Symbol.dispose> & {
```

Comment:
what is Symbol.dispose?

Reply:
Generated by core crypto wasm build generator, CC team asked to ignore it just like the "free" property